source/clear-linux/get-started/install-configfile.rst

```diff
@@ -0,0 +1,119 @@
+.. _install-configfile:
+
+Install using clr-installer and a configuration file
+####################################################
+
+This guide shows you how to install |CL-attr| using the clr-installer tool
+with a configuration file. This method allows you to predefine configurations
+and options, which are then fed into clr-installer, thus relieving you of
+having to walk through the installer step by step. You can save the
+configuration file and reuse it again.
+
+System requirements
+*******************
+
+Ensure that your target system supports the installation:
+
+* :ref:`system-requirements`
+* :ref:`compatibility-check`
+
+Process
+*******
+
+Perform the following steps:
+
+#. Download the latest :file:`live-server.iso` file.
+
+   For example:
+   https://download.clearlinux.org/releases/29690/clear/clear-29690-live-server.iso.xz
+
+#. Uncompress it and burn it to a USB thumb drive.
+#. Boot up the USB thumb drive.
+#. Select :guilabel:`Clear Linux OS` from the menu.
+#. After it boots, press :guilabel:`<CTRL><Shift><F2>` to bring up a TTY console.
+#. Log in as root and set a password.
+#. Verify you have an IP address (network connection) and configure proxies (if needed).
+#. Download an :file:`installer.yaml` template.
+
+   For example:
+
+   .. code-block:: bash
+
+      # curl -O https://download.clearlinux.org/releases/29690/clear/config/image/installer.yaml

```

Comment:
The file `installer.yaml` is no longer published in later versions. See for example: 
https://download.clearlinux.org/releases/30010/clear/config/image/

However, a similar file is published at link above: `live-server.yaml`
It shows a slightly different template than the example here.